src/en/models-defining.md

```diff
@@ -1,30 +1,42 @@
 Title: Defining a Juju Model
 TODO: Review again soon (created: March 2016)
-      Remove bug citation (1552469) when fixed and remove --config option from example command
+      Remove Note re bug 1552469 when fixed; remove Note and --config option from Rackspace example
 
 
 # Defining a model
 
 Use the `juju create-model` command to create a *hosted model* for a given
-cloud provider:
+provider:
```

Comment:
is it for a 'given provider'? I thought it was for the current controller. The model couldn't care less where it actually is, surely